seldon/seldon-core-operator/base/resources.yaml

```diff
@@ -3440,7 +3440,7 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
+            memory: 300Mi
```

Comment:
Should this change also go in our overlays?
Also, not that it's super critical, but is 300Mi the lowest "good" value?  ie:  is 100Mi sufficient?  200Mi?

Reply:
We had it 300 before, and it was not in an overlay before, I dont think it should be since it is not OCP specific

Reply:
per discussion, this is actually a change tbd upstream as well, so it doesn't belong in our overlay